src/main/resources/avro/referencemethods.avdl

```diff
@@ -166,17 +172,17 @@ Reference getReference(
     */
     string id) throws GAException;
 
-/****************  /references/{id}/bases  *******************/
```

Comment:
I think removing this API call will cause major compatibility headaches as we try to merge the graph capable API back into the main repo. I would suggest we have a new interface that we use for getting the sequence for a segment/set of segments, and keep the `ListReferenceBases` interface intact. We can then aim for a slow deprecation process in favour of the graph API (which will natively support a linear reference).